bitmap-raziyehLogan/lib/bitmap-file-write.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var fs = require('fs');
+
+module.exports = function(outputfileName, buffer, callback){
+  fs.writeFile(`${__dirname}/../data/${outputfileName}.bmp`,buffer, function(err, data) {
```

Comment:
@loganlsr @Raziyehbazargan - I really like that you are passing in the buffer here - also, great use of string interpolation for the output filename string